test/test.js

```diff
@@ -179,6 +179,24 @@ describe('async-helpers', function() {
         assert.deepEqual(val, 'doowb');
       });
     });
+
+    it('should support sync and async value items in argument arrays', () => {
```

Comment:
Might not be pretty clear what it does from the title...

But basically tests if array argument passed to helper, can have actual values and asyncIds which will be resolved to actual values.